packages/components/pie-button/test/component/pie-button.spec.ts

```diff
@@ -1,228 +1,127 @@
-import { getShadowElementStylePropValues } from '@justeattakeaway/pie-webc-testing/src/helpers/get-shadow-element-style-prop-values.ts';
-import { test, expect } from '@sand4rt/experimental-ct-web';
-import { PieButton, type ButtonProps } from '../../src/index.ts';
+import { test, expect } from '@playwright/test';
+import { ButtonComponent } from '../helpers/page-object/pie-button.page.ts';
```

Comment:
I think this is quite a misleading name. This is a page object, not a component, right? Would `ButtonComponentPage` or something similar work?